Sources/OAuth2/AuthProviders/Vk.swift

```diff
@@ -27,22 +27,22 @@ import PerfectSession
 
 /// Vk configuration singleton
 public struct VkConfig {
-
+    
```

Comment:
Please remove these extra/trailing whitespaces. Your Xcode settings may need to be adjusted to automatically trim them.
<img width="406" alt="screen shot 2018-03-05 at 11 15 00 am" src="https://user-images.githubusercontent.com/51949/36994610-813ace28-2066-11e8-9fb1-64e5fc573cf7.png">